### PR TITLE
v1.9.0

### PR DESCRIPTION
**Pull request by Anush008**
## Description
Heading to `v1.9`.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # The version of qdrant to use to download protos
-qdrantProtosVersion=v1.8.0
+qdrantProtosVersion=dev
 
 # The version of qdrant docker image to run integration tests against
-qdrantVersion=v1.8.0
+qdrantVersion=dev
```
Comment (Anush008) [inline gradle.properties:5]: TODO: Switch to `v1.9.0` when out.

File: src/test/java/io/qdrant/client/ApiKeyTest.java [hunk 2]
```diff
@@ -61,7 +61,8 @@ public void client_without_api_key_cannot_connect() {
 			Throwable cause = executionException.getCause();
 			assertEquals(StatusRuntimeException.class, cause.getClass());
 			StatusRuntimeException statusRuntimeException = (StatusRuntimeException) cause;
-			assertEquals(Status.Code.PERMISSION_DENIED, statusRuntimeException.getStatus().getCode());
+			// FIXME: This assertion fails because the server returns UNAUTHENTICATED instead of PERMISSION_DENIED
+			// assertEquals(Status.Code.PERMISSION_DENIED, statusRuntimeException.getStatus().getCode());
```
Comment (russcam) [inline src/test/java/io/qdrant/client/ApiKeyTest.java:65]: Recommend annotating method with `@Disabled("<reason>")` as it's more discoverable

**Review by russcam — APPROVED** [review]

File: src/test/java/io/qdrant/client/ApiKeyTest.java (hunk 2 above)
Comment (Anush008) [reply src/test/java/io/qdrant/client/ApiKeyTest.java:65]: 👍👍

Will switch the check to `Status.Code.UNAUTHENTICATED` once the `1.9` image is out.

Comment (timvisee) [reply src/test/java/io/qdrant/client/ApiKeyTest.java:65]: @Anush008 could you arrange this one? The image should be available now.

File: gradle.properties (hunk 1 above)
Comment (timvisee) [reply gradle.properties]: Done in 970739388fec958bb7e16fc7e19d2d1688cd1d49

File: src/test/java/io/qdrant/client/ApiKeyTest.java (hunk 2 above)
Comment (Anush008) [reply src/test/java/io/qdrant/client/ApiKeyTest.java:65]: Will do.

**Pull request merged**